specs/README.md

```diff
@@ -0,0 +1,54 @@
+# Project Reunion Specs
+
+This directory contains archived and in-progress spec documents for APIs in
+Project Reunion.
+
+Documents in this directory are used as part of a feature and API reiview
+for features in development and are not intended to be final developer
```

Comment:
No, but they will be used as the primary source material to generate such docs. So they should be written with the final docs in mind and with developers as the audience. That's what we've been steering towards for WinUI and I think it's been a great change from API specs of old.